src/app/components/navigation/Navbar.js

```diff
@@ -1,90 +1,75 @@
+"use client";
+import React from "react";
 import {
```

Comment:
I'm not sure what these changes are, but if they are independent of the alumni page, they should be in a separate PR